projects/js-packages/connection/components/manage-connection-dialog/index.jsx

```diff
@@ -117,18 +195,30 @@ const ManageConnectionDialog = props => {
 	);
 };
 
-const ManageConnectionActionCard = ( { title, onClick = () => null, link = '#', action } ) => {
+const ManageConnectionActionCard = ( {
+	title,
+	onClick = () => null,
+	link = '#',
+	action,
+	disabled,
```

Comment:
Should we have disabled be `false` by default? I guess an omission leaving it null would most likely have the same behavior, I guess I just prefer explicit defaults 😅